Ubuntu 18.04 fixes for installer script

wanxewoj opened the pull request:
This is a suggested implementation for [Feature request: Ubuntu 18.04 support for the install script](https://github.com/zhiverbox/armbian-build/issues/1), which handles the differences between 16.04 (xenial) and18.04 (bionic).

wanxewoj committed:
Ubuntu 18.04 fixes for installer script

This is a suggested implementation for the `Installer` https://github.com/zhiverbox/armbian-build/issues/1, which handles the differences between 16.04 (xenial) and18.04 (bionic).

zciendor commented:
Thanks a lot for your contribution! The changes look OK, however you included quite a few non-bionic changes which bloat this pull request an make it harder to review. Please open more `atomic` pull requests in the future.

One comment though:
The `exit 1` in `check_host_os()` seems a bit harsh. Could we instead show a warning, that the host OS in untested and the user will likely face errors if he continues? This allows that we get feedback from other host OSes and versions. Eventually they work out of the box or are simple to fix/support...

wanxewoj committed:
allow to continue on unsupported systems

User can continue the installer on unsupported host operating systems by choosing a host OS profile.

wanxewoj commented:
Reflected your comment in the latest commit. Users can now continue on unsupported systems by selecting the profile (xenial or bionic) to use.
